org.civicrm.bootstrapcivicrm/scss/civicrm/contact/pages/_manage-groups.scss

```diff
@@ -1,3 +1,101 @@
+.page-civicrm-contact {
+  #GroupContact {
+
+    h3 {
+      padding: 20px;
+      margin: 0;
+      background: $gray-lighter;
+      color: $gray-darker;
+      font-size: $font-size-h2;
+      font-family: $font-family-base;
+      border-bottom: 1px solid $crm-background;
+    }
+
+    table {
+      background: transparent;
+      border: none;
+      font-family: $font-family-base;
+
+      thead {
+
+        tr{
```

Comment:
Let's save some nesting selectors. Can we do 
`thead tr {...}` 
rather than 
`thead { tr {...} }` ?